src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplication.groovy

```diff
@@ -64,11 +64,11 @@ class DockerJavaApplication {
     final ListProperty<Integer> ports
 
     /**
-     * The tag used for the Docker image.
+     * The tags used for the Docker image.
      * <p>
-     * Defaults to {@code <project.group>/<applicationName>:<project.version>}.
+     * Defaults to {@code [<project.group>/<applicationName>:<project.version>]}.
      */
-    final Property<String> tag
+    final ListProperty<String> tags
```

Comment:
We should document here that **only** the first tag in the list will be pushed.

Reply:
Should we iterate over all potential tags and attempt to push? Maybe have `DockerPushImage` also take in a list of tags and we attempt to push each?

Reply:
@cdancy I do like that idea in case we want to move forward with it.